tests-integration/spark/model/src/SparkTests/FunctionTests.elm

```diff
@@ -151,6 +159,30 @@ testFilter2 source =
     source |> List.filter (filterFnWithVar 17)
 
 
+testListMinimum : List Antique -> List { min : Maybe Int }
```

Comment:
See comment in the .md file.  We also want to support the idiomatic Elm way to calculate a max/min.
``` Elm
source
  |> List.map mappingFunction
  |> List.maximum
```